src/wasm/ngx_wasm_ops.c

```diff
@@ -126,7 +126,7 @@ ngx_wasm_ops_plan_load(ngx_wasm_ops_plan_t *plan, ngx_log_t *log)
     ngx_uint_t               *fid;
     ngx_array_t              *ids;
     ngx_wasm_op_t            *op;
-    ngx_wasm_ops_pipeline_t  *pipeline;
+    ngx_wasm_ops_pipeline_t  *pipeline = NULL;
```

Comment:
May I suggest making a separate commit for the `src`/ changes? That way they will appear in the changelog as fix for newer compiler versions!

Reply:
Thank you for pointing that out! I'll make a separate commit for the src/ changes.